doc/python/px-arguments.md

```diff
@@ -33,19 +33,33 @@ jupyter:
     thumbnail: thumbnail/plotly-express.png
 ---
 
-### Plotly Express works with Column-oriented, Matrix or Geographic Data
+Plotly Express (`px`) is the high-level interface to Plotly and provides functions for generating charts. `px` functions support data provided in a number of different formats (long, wide, and mixed) and as different types of objects, including pandas and Polars DataFrames.
 
-[Plotly Express](/python/plotly-express/) is the easy-to-use, high-level interface to Plotly, which [operates on a variety of types of data](/python/px-arguments/) and produces [easy-to-style figures](/python/styling-plotly-express/).
+## Data for the Examples
 
-Plotly Express provides functions to visualize a variety of types of data. Most functions such as `px.bar` or `px.scatter` expect to operate on column-oriented data of the type you might store in a `DataFrame` (in either "long" or "wide" format, see below). These functions use Pandas internally to process the data, but also accept other types of DataFrames as arguments. See the **Input Data as Non-Pandas DataFrames** section below for more details.
+The examples on this page use datasets available in the `data` package in `px`. `px.data` contains functions that when called return a dataset as a DataFrame. Some of the datasets included in `px.data` are:
 
-[`px.imshow` operates on matrix-like data](/python/imshow/) you might store in a `numpy` or `xarray` array and functions like [`px.choropleth` and `px.choropleth_mapbox` can operate on geographic data](/python/maps/) of the kind you might store in a GeoPandas `GeoDataFrame`. This page details how to provide column-oriented data to most Plotly Express functions.
+- `carshare` - Each row represents the availability of car-sharing services near the centroid of a zone in Montreal over a month-long period.
+- `election` - Each row represents voting results for an electoral district in the 2013 Montreal mayoral election.
+- `iris` - Each row represents a flower.
 
+To access the `iris` dataset, we call it's function and assign it to a variable:
 
+```python
+import plotly.express as px
+
+df = px.data.iris()
+df.head()
+```
+
+By default `px.data` functions return Pandas DataFrames, but you can specify an alternative DataFrame type using `return_type`. `pandas`, `polars`, and `pyarrow` are supported return types.
 
-### Plotly Express works with  Long-, Wide-, and Mixed-Form Data
+```python
+df = px.data.iris(return_type='polars')
+df.head()
+```
 
-*Until version 4.8, Plotly Express only operated on long-form (previously called "tidy") data, but [now accepts wide-form and mixed-form data](/python/wide-form/) as well.*
+## Long, Wide-, and Mixed-Form Data
```

Comment:
puzzled: why no dash after Long when there is one after Wide? (I wouldn't have a dash after either.)